requirements.txt

```diff
@@ -4,28 +4,26 @@ SCons==4.5.2
 # NVDA's runtime dependencies
 comtypes==1.2.0
 pyserial==3.5
-wxPython==4.1.1
-git+https://github.com/DiffSK/configobj@3e2f4cc#egg=configobj
+wxPython==4.2.1
+git+https://github.com/DiffSK/configobj@e2ba445#egg=configobj

```

Comment:
While at it could you please use full hash of the commit? If incomplete hash is used pip shows the following warning:
```
  WARNING: Did not find branch or tag '3e2f4cc', assuming revision or ref.
```

Reply:
fixed